specs/operator/addon.spec

```diff
@@ -63,4 +63,15 @@ Importance: Critical
 Pos/Neg: Negative
 
 Steps:
-  * Update addon config with resolverTasks as "false" and pipelineTemplates as "true" and expect message "validation failed: pipelineTemplates cannot be true if resolverTask is false"
\ No newline at end of file
+  * Update addon config with resolverTasks as "false" and pipelineTemplates as "true" and expect message "validation failed: pipelineTemplates cannot be true if resolverTask is false"
+
+## Verify versioned ecosystem tasks: PIPELINES-15-TC09
+Tags: e2e, integration, admin, addon, sanity
```

Comment:
this doesn't need to run as admin, remove sanity

Reply:
Done